Tests/PackageRegistryTests/RegistryClientTests.swift

```diff
@@ -1906,18 +1755,11 @@ final class RegistryClientTests: XCTestCase {
             }
 
             let parsedToolsVersion = try ToolsVersionParser.parse(utf8String: manifest)
-            XCTAssertEqual(parsedToolsVersion, .v5_3)
+            #expect(parsedToolsVersion == toolsVersion ?? .current)
```

Comment:
issue (blocking): the `testDiagnostics(...)` calls from a few lines before is XCTest-specific as it has an `XCFail(...)` call. Use `expectDiagnostic(...)` instead.

Reply:
Done